lib/routes/stbu/jsjxy.ts

```diff
@@ -15,17 +15,18 @@ export const route: Route = {
     features: {
         requireConfig: false,
         requirePuppeteer: false,
-        antiCrawler: false,
+        antiCrawler: true,
         supportBT: false,
         supportPodcast: false,
         supportScihub: false,
     },
     radar: [
         {
-            source: ['jsjxy.stbu.edu.cn/news', 'jsjxy.stbu.edu.cn/'],
+            source: ['jsjxy.stbu.edu.cn/news', 'jsjxy.stbu.edu.cn', 'stbu.edu.cn'],
+            target: '/stbu/jsjxy',
```

Comment:
If `target` is not specified, it will be the same as `path`, and the Radar output will be `/stbu/jsjxy` since the namespace will be automatically prepended during build. However, if `target` is set to `/stbu/jsjxy`, the Radar output will be `/stbu/stbu/jsjxy`.